tests/FSharpPlusFable.Tests/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "scripts": {
+    "buildFSharpPlus": "fable-splitter ../../src/FSharpPlus -o FSharpPlusFable --commonjs --allFiles",
```

Comment:
@wallymathieu This is the local target I've added to the local build file of the F#+ Fable tests.